bundles/CoreBundle/DependencyInjection/Configuration.php

```diff
@@ -999,7 +999,7 @@ private function addDocumentsNode(ArrayNodeDefinition $rootNode)
                 ->end()
             ->end();
 
-        $this->addImplementationLoaderNode($documentsNode, 'type_definitions');
+        $this->addClassResolverNode($documentsNode, 'type_definitions');
```

Comment:
Not sure if this would be a BC break
https://github.com/pimcore/pimcore/pull/14509/commits/bebf533a764932c74dd39a5f8d67f5a4b87a1f62
if one had previously set some custom prefixes, it would now throw
`Unrecognized option "prefixes" under "pimcore.documents.type_definitions`

Reply:
This was introduced in 10.6 and never finalized so I would not consider this a BC.